internal/webhook/spinapp_validating.go

```diff
@@ -104,10 +104,7 @@ func validateExecutor(spec spinv1alpha1.SpinAppSpec, fetchExecutor func(name str
 }
 
 func validateReplicas(spec spinv1alpha1.SpinAppSpec) *field.Error {
-	if spec.EnableAutoscaling && spec.Replicas != 0 {
-		return field.Invalid(field.NewPath("spec").Child("replicas"), spec.Replicas, "replicas cannot be set when autoscaling is enabled")
-	}
-	if !spec.EnableAutoscaling && spec.Replicas < 1 {
+	if spec.Replicas < 1 {
```

Comment:
I think I'd be ok with Replicas being set to 0 now that you can scale up to `1` later - it would enable some potentially interesting loadbalancer integrations.